src/games/strategy/engine/framework/headlessGameServer/HeadlessServerSetup.java

```diff
@@ -183,4 +183,10 @@ public void postStartGame() {
     final GameData data = m_gameSelectorModel.getGameData();
     data.getProperties().set(PBEMMessagePoster.PBEM_GAME_PROP_NAME, false);
   }
+
+  @Override
+  public boolean isMetaSetupPanelInstance() {
+    return false;
```

Comment:
Would it be better to add this method to SetupPanel with returning false and just override it in MetaSetupPanel rather than implementing in all the classes?